lib/routes/lnurl/payment/lnurl_payment_page.dart

```diff
@@ -129,10 +109,14 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
                     controller: _amountController,
                     validatorFn: validatePayment,
                   ),
-                  ReceivableBTCBox(
-                    receiveLabel: texts.lnurl_fetch_invoice_limit(
-                      (widget.requestData.minSendable ~/ 1000).toString(),
-                      (widget.requestData.maxSendable ~/ 1000).toString(),
+                  Padding(
+                    padding: const EdgeInsets.only(top: 8),
+                    child: Text(
+                      texts.lnurl_fetch_invoice_limit(
+                        (widget.requestData.minSendable ~/ 1000).toString(),
+                        (widget.requestData.maxSendable ~/ 1000).toString(),
+                      ),
+                      textAlign: TextAlign.left,
```

Comment:
Please use the same text style with Breezmobile UI.
Breezmobile text theme values are likely to be deprecated and replaced with updated versions on c-Breez. Ping me in Slack to find the equivalent theme value.